python/ray/tests/test_metrics_agent.py

```diff
@@ -110,10 +112,49 @@
 # This list of metrics should be kept in sync with
 # ray/python/ray/autoscaler/_private/prom_metrics.py
 _DASHBOARD_METRICS = [
-    "dashboard_api_requests_duration_seconds",
-    "dashboard_api_requests_count",
+    "ray_dashboard_api_requests_duration_seconds",
+    "ray_dashboard_api_requests_count",
+    "ray_component_cpu",
+    "ray_component_rss",
 ]
 
+_NODE_METRICS = [
+    "ray_node_cpu_utilization",
+    "ray_node_cpu_count",
+    "ray_node_mem_used",
+    "ray_node_mem_available",
+    "ray_node_mem_total",
+    "ray_node_disk_io_read",
+    "ray_node_disk_io_write",
+    "ray_node_disk_io_read_count",
+    "ray_node_disk_io_write_count",
+    "ray_node_disk_io_read_speed",
+    "ray_node_disk_io_write_speed",
+    "ray_node_disk_read_iops",
+    "ray_node_disk_write_iops",
+    "ray_node_disk_usage",
+    "ray_node_disk_free",
+    "ray_node_disk_utilization_percentage",
+    "ray_node_network_sent",
+    "ray_node_network_received",
+    "ray_node_network_send_speed",
+    "ray_node_network_receive_speed",
+]
+
+_NODE_COMPONENT_METRICS = [
+    "ray_component_cpu",
+    "ray_component_rss",
+    "ray_component_uss",
+]
+
+if sys.platform == "linux":
+    # Below metrics are only available in Linux.
+    _NODE_COMPONENT_METRICS += [
```

Comment:
How about we don't report these for now? We'd probably avoid them in the dashboard anyways since they aren't available all platforms.